scripts/Startslide_Right.py

```diff
@@ -0,0 +1,55 @@
+from dolphin import event, gui, utils
+from Modules import TTK_Lib
+from Modules import mkw_classes as classes
+from Modules import mkw_core as core
+from Modules import mkw_translations as translate
+from Modules.framesequence import FrameSequence
+import os
+
+flame_slide_bikes = ("Flame Runner", "Mach Bike", "Sugarscoot", "Zip Zip")
+spear_slide_bikes = ("Jet Bubble", "Phantom", "Spear", "Sneakster", "Wario Bike")
+star_slide_bikes  = ("Bit Bike", "Bullet Bike", "Dolphin Dasher", "Magikruiser",
```

Comment:
Was just curious, is there any performance different using a tuple here instead of a list?

Reply:
I doubt the performance is significantly different whether a tuple was used or not, I thought I might as well use one here in case it did.